### ISSUE TITLE
install: add instructions to update/uninstall DVC

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/dvc.org#702]
At least for the recommended install methods for each platform

> First mentione in https://github.com/iterative/dvc.org/pull/695#pullrequestreview-302330591

**Comment by dashohoxha** [comment]
@jorgeorpinel Don't you think that the user should know how to use his commands (`apt`, `yum`, `pip`, etc.)? So, do we need to tell him how to update/upgrade a package?

I think that the same argument goes for telling users when and how to use a virtualenv. They should know when and how to use it, it should not be our concern. But this is a different topic.

**Comment by jorgeorpinel** [comment]
Yes, it's a good point @dashohoxha. But at least for `pip`, since upgrading can be different if you install from PyPi vs. if you install from local code, maybe we should include the commands in our docs.`

What do you think @shcheklein?

**Comment by jorgeorpinel** [comment]
On second thought I don't think my argument agains teaching other technologies really applies to installing and upgrading DVC, even if it's done with a 3rd party tool.

In fact maybe we should even include instructions to **uninstall** DVC as well somewhere. (Other similar tools offer this kind of information, probably because users look for it e.g https://devcenter.heroku.com/articles/heroku-cli#uninstalling-the-heroku-cli.)

Also, what should be the right term be: "update" or "upgrade"? 😋

**Comment by algomaster99** [comment]
@jorgeorpinel oh-my-zsh also does this. I think we should also have a command like `uninstall_dvc` which removes dvc installed from source or binary. Otherwise, there will be many separate instructions to uninstall it. If you think this is valid, please make it a separate issue.

**Comment by jorgeorpinel** [comment]
Not a bad idea. Opened iterative/dvc/issues/2637 in parallel.

The question is should we provide instructions in the docs for now?

**Comment by shcheklein** [comment]
@jorgeorpinel nice, but not a top priority I would say. Uninstall can be really useful in some case (Mac OS package for example).

**Comment by jorgeorpinel** [comment]
From https://github.com/iterative/dvc.org/issues/65#issuecomment-504713595:

Related discussion: https://discordapp.com/channels/485586884165107732/485596304961962003/574941227624169492

For Mac (installed from pkg):

```
$ which dvc
/usr/local/bin/dvc -> /usr/local/lib/dvc/dvc

$ ls -la /usr/local/bin/dvc
/usr/local/bin/dvc -> /usr/local/lib/dvc/dvc

sudo rm -f /usr/local/bin/dvc
sudo rm -rf /usr/local/lib/dvc
sudo pkgutil --forget com.iterative.dvc
```

**Comment by efiop** [comment]
For the record: need to write down how to upgrade with dependencies: https://github.com/iterative/dvc/issues/4522#issuecomment-688095901

**Comment by jorgeorpinel** [comment]
> Uninstall can be really useful in some case (Mac OS package for example).

This case is already documented. Any other cases where we may want to document update/uninstall vs letting the users figuring it out? Cc @shcheklein @dberenbaum 

Closing as stale for now
